js/src/components/multiline.component.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-bitwise */
```

Comment:
we should disable rules only per specific line, see https://stackoverflow.com/questions/27732209/turning-off-eslint-rule-for-a-specific-line